runtime/syntax/pascal.vim

```diff
@@ -146,7 +146,7 @@ if !exists("pascal_no_functions")
   " ordinal functions
   syn keyword pascalFunction	Odd Pred Succ
 
-  " transfert functions
+  " conversion functions
   syn keyword pascalFunction	Chr Ord Round Trunc
```

Comment:
I've already fixed this in my local copy and will send it to Bram with the next update.

FWIW, these are actually known as transfer functions in Pascal world.